jetty-websocket/jetty-websocket-server/src/main/java/module-info.java

```diff
@@ -28,6 +28,8 @@
 
     requires javax.servlet.api;
     requires org.eclipse.jetty.util;
+    requires org.eclipse.jetty.http;
+    requires org.eclipse.jetty.server;
```

Comment:
You'll need `org.eclipse.jetty.jmx` if you add `org.eclipse.jetty.server` as a static reference (i think)

Reply:
@joakime I don't doubt you... but @sbordet can you explain what we should be doing with jmx in module infos?

Reply:
Module `org.eclipse.jetty.jmx` should typically be referenced as `static`.
The question you should ask is: "is this module working if `org.eclipse.jetty.jmx` is not loaded by the JVM?".
If the answer is "yes", then add `static` to the `requires` statement.

If you need to _provide_ JMX MBeans, then look at how's done in `org.eclipse.jetty.client`.
It's typically something like: `exports org.eclipse.jetty.<module>.jmx to org.eclipse.jetty.jmx`, so that the JMX module can "see" (and be able to load and instantiate) the JMX classes this module provides.

Reply:
I can't say I really understand why a websocket module must declare jmx as a static requirement just because it knows about jetty-server?  But I have added the magic line, said a few voodoo chants and slaughtered a goat... so perhaps this is correct.   Do we have to do this everywhere now?

Reply:
Just the dependency on `jetty-server` should not require `jetty-jmx`. If that's so, then `jetty-server` module-info.java is wrong.

Reply:
If, OTOH, the websocket module defines some custom JMX (e.g. extends `ObjectMBean` or such), then it should be a `requires static`.

Reply:
I'll leave as is, but I think we need to do a check externally to this issue to ensure the module-info is correct for JMX everywhere.